examples/next/.gitignore

```diff
@@ -33,5 +33,4 @@ yarn-error.log*
 
 # amplify
 **/.amplify
-**/amplify_outputs*
```

Comment:
There are amplify_outputs files in the e2e folder

Reply:
Oh wait we do want to be able to keep those comitted, disregard 😅

Reply:
This is just in the examples directory so that we can pull the outputs file from environments